pkg/cmd/scylla-manager/db.go

```diff
@@ -58,7 +58,8 @@ func createKeyspace(ctx context.Context, c config.Config, logger log.Logger) err
 	return session.Query(mustEvaluateCreateKeyspaceStmt(c)).Exec()
 }
 
-const createKeyspaceStmt = "CREATE KEYSPACE {{.Keyspace}} WITH replication = {'class': 'SimpleStrategy', 'replication_factor': {{.ReplicationFactor}}}"
+const createKeyspaceStmt = "CREATE KEYSPACE {{.Keyspace}} WITH replication = {'class': 'SimpleStrategy', 'replication_factor': {{.ReplicationFactor}}}" +
+	" AND tablets = {'enabled': false}"
```

Comment:
This unfortunately won't work with 2024.1, which is currently still supported by SM.
For this scylla version, we can't use the `tablets = {'enabled': false}` syntax.
Tests are passing because we don't have real e2e tests in SM repo, and those tests use different functions for creating SM keyspace. If we were to execute SCT/DTests with this change and scylla 2024.1, we would fail on SM startup.